requirements/index.html

```diff
@@ -44,6 +44,51 @@ <h2>Use Cases</h2>
 
   <section id="requirements">
     <h2>Requirements</h2>
+    <section id="profile-registry-entries-requirements">
+      <h3>Profile Registry Entries</h3>
+      <ol>
+        <li>Each Profile specification shall specify an identifier (a stable URI) which identifies the profile</li>
+        <li>Profiles may constrain a finite set of protocol bindings that a conformant Thing may use and that a
+          conformant Consumer can be expected to support
+          <ul>
+            <li>
+              Profiles may constrain conformant Things to always follow certain defaults defined in a protocol binding
+              document
+            </li>
+          </ul>
+        </li>
+        <li>Profiles may constrain a finite set of payload bindings that a conformant Thing may use and that a
+          conformant Consumer can be expected to support
+          <ul>
+            <li>Profiles may constrain conformant Things to always follow certain default payload formats defined in a
+              payload binding document</li>
+          </ul>
+        </li>
+        <li>Profiles may constrain a finite set of security schemes that a conformant Thing may use and that a
+          conformant Consumer can be expected to support</li>
+        <li>Profiles may constrain a finite set of discovery mechanisms that a conformant Thing may use and that a
+          conformant Consumer can be expected to support</li>
+        <li>Profiles may constrain a finite set of link relation types that a conformant Thing is recommended to use,
+          and how they should be interpreted by a WoT Consumer</li>
+        <li>Profiles may constrain a finite set of semantic contexts that a conformant Thing is recommended to use and
+          that a conformant Consumer can be expected to support</li>
+        <li>Profiles shall not directly extend the WoT Thing Description information model, which should be defined
+          using TD context extensions</li>
+        <li>Profiles shall not define concrete protocol bindings, which should be defined as defaults in a protocol
+          binding document and referenced from a Profile instead</li>
+        <li>Profiles shall not define data payload formats, which should be specified in a payload binding document</li>
+        <li>Profiles shall not define new discovery mechanisms, which should be defined in the WoT Discovery
+          specification</li>
```

Comment:
Is there no extension mechanism? How does the WoT Discovery specification evolve to include "new discovery mechanisms"?
```suggestion
        <li>Profiles shall not define new discovery mechanisms, which should instead be defined in the WoT
          Discovery specification.</li>
```